spec/integration/integration_helper.rb

```diff
@@ -215,8 +215,8 @@ def index?(table_name, key_name, type = :non_unique)
   # Environment
   #
 
-  def master_slave_mode?
-    !!ENV['MASTER_SLAVE']
+  def master_replica_mode?
```

Comment:
should we also rename `master` to `main` or something?

Reply:
Yeah, we can do something like that on a follow-up PR.